### ISSUE TITLE
Conclusion missing for Part2

### ISSUE THREAD

**Issue by JagaRamesh** [deepjhavar/intro-data-capstone-biodiversity#1]
In "Foot and mouth disease analysis", the sample size is determined but the following questions are unanswered
1. How many weeks would you need to observe sheep at Bryce National Park in order to observe enough sheep? 
2. How many weeks would you need to observe at Yellowstone National Park to observe enough sheep?

**Comment by deepjhavar** [comment]
thank you , There is one doubt i had , so the exercise on code academy , i can give the answer for yellow stone national park that it would take one week to observe but with  bryce i am confused , There were only 250 observations so i would assume half a week but on code academy when i do 510/250 it gives the correct answer. So i wanted to ask you what is the correct conclusion.Th

**Comment by deepjhavar** [comment]
Also should i resubmit ?

**Comment by JagaRamesh** [comment]
For Bryce National Park, it would take ~2 weeks (510/250) to observe enough sheep.

**Comment by JagaRamesh** [comment]
I don't believe, you need to resubmit the project. Implement the recommendations highlighted in review and then you can close the issue.

**Comment by deepjhavar** [comment]
I have implemented , But should this be sufficent for my certificate ? then i can close the issue
